### ISSUE TITLE
Update ATO docs and corresponding code docs 

### ISSUE THREAD

**Issue by lfrohlich** [raft-tech/TANF-app#962]
Follow up action items from ATO debrief: 

- Configuration Management Plan

- [x] Need clarity on what does the app maintain vs. cloud.gov
- [x] What is the maintenance freq? Need more details in Section 2.4 about how a secure state will be maintained after ATO (dependabot)

- SSP

- [x] p. 12 define privileged user
- [x] Replace links with content
- [x] 2.11.3 clarify authentication for OFA admin and other users
- [x] p. 12-13 Workflow request approval and creation, clarify how the request comes in and who is authorized to approve the request. Need to define that the SO is approving admin access.
- [ ] Section 2.14.1 Get IAA from Sandra's team about their login.gov process as a sample to work from here
- [x] Section 2.11.6 ACF Access Management SOP (Penyin)

Update code docs to reflect ATO docs changes: 

- [ ] User role management https://github.com/raft-tech/TANF-app/blob/user-role-edits/docs/Technical-Documentation/user_role_management.md
- [x] Dependabot
- [ ] Buildpacks
- [ ] security control docs per SAR

**Comment by amilash** [comment]
Is this ready to be closed, or is there more work to do?

**Comment by ADPennington** [comment]
> Is this ready to be closed, or is there more work to do?

@amilash more work before I can update this:
- we're still waiting on SAR from OCIO. @lfrohlich do you think we should continue holding?
- the screenshots for user profile need to be updated (i can do this after next part is completed) 
- (STT) data prepper user group needs to be changed to data analyst (if we're created all the necessary user groups as part of the permission epic, could include this task as part of one of the permission tickets).

**Comment by lfrohlich** [comment]
@ADPennington  we could do the other things and then create a separate ticket for SAR updates once we get teh SAR

**Comment by amilash** [comment]
Is this done and we can close @lfrohlich

**Comment by lfrohlich** [comment]
I'll close and once (if) we get the SAR we can update those docs
